testutil/lowlevel_test.go

```diff
@@ -65,6 +65,9 @@ func (s *lowLevelSuite) TestOpenSuccess(c *C) {
 	c.Assert(s.sys.Calls(), DeepEquals, []string{
 		`open "/some/path" O_NOFOLLOW|O_CLOEXEC|O_RDWR|O_CREAT|O_EXCL 0`, // -> 3
 	})
+	c.Assert(s.sys.RCalls(), DeepEquals, []testutil.CallResultError{
```

Comment:
Why do we need both s.sys.Calls() and RCalls() ? That looks a bit redundant, no? Same question to the similar additions below.

Reply:
Here to ensure I didn't break anything. I plan to deprecate Calls but there's lots of users still.

Reply:
Thanks, maybe a small comment in Calls() that it should not be used and that RCalls is the future (at your leisure).

Reply:
I'll work tirelessly to drop all calls to Calls() and then the function itself.